src/VisualStudio/Core/Def/Implementation/AbstractVsTextViewFilter`3.cs

```diff
@@ -64,7 +69,7 @@ public int GetPairExtents(int iLine, int iIndex, TextSpan[] pSpan)
         {
             int result = VSConstants.S_OK;
             _languageService.Package.ComponentModel.GetService<IWaitIndicator>().Wait(
-                "Intellisense",
+                EditorFeaturesResources.Intellisense,
```

Comment:
Note that "Intellisense" is actually correctly spelled "IntelliSense". The editor APIs got their namespace wrong.